react-components/src/components/CacheProvider/FdmNodeCache.ts

```diff
@@ -228,16 +224,16 @@ export class FdmNodeCache {
   private async getAndCacheRevisionToEdgesMap(
```

Comment:
Maybe it is better to change any `RevisionToEdgesMap` wording to something like `RevisionToConnectionsMap` as well